src/main/java/net/consensys/orion/config/Config.java

```diff
@@ -659,6 +683,73 @@ private static Schema configSchema() {
         "TLS known servers for the client. This contains the fingerprints of public keys of other nodes that this node has encountered for the ca-or-tofu, tofu and whitelist trust modes.",
         null);
 
+    schemaBuilder.addString(
+        "clientconnectiontls",
+        "off",
+        "TLS status. Options:\n"
+            + "\n"
+            + "   - strict: All connections to the client connection of this node must use TLS with client authentication\n"
+            + "       See the documentation 'clientconnectiontlsservertrust'\n"
+            + "   - off: Mutually authenticated TLS is not used for in- and outbound\n"
+            + "       connections, although unauthenticated connections to HTTPS hosts are still possible. This\n"
+            + "       should only be used if another transport security mechanism like WireGuard is in place.",
+        PropertyValidator.anyOfIgnoreCase("off", "strict"));
+
+    schemaBuilder.addString(
+        "clientconnectiontlsservertrust",
+        "tofu",
+        "TLS trust mode for the internal server endpoint. This decides who's allowed to connect to it. Options:\n"
+            + "\n"
+            + "   - whitelist: Only nodes presenting certificates with fingerprints in 'tlsknownclients'\n"
+            + "       will be allowed to connect.\n"
+            + "   - ca: Only nodes with a valid certificate and chain of trust to one of the\n"
+            + "       system root certificates will be allowed to connect. The folder containing trusted root\n"
+            + "       certificates can be overridden with the SYSTEM_CERTIFICATE_PATH environment variable.\n"
+            + "   - insecure-tofa: (Trust-on-first-access) On first connection to this server the common name\n"
+            + "       and fingerprint of the presented certificate will be added to 'tlsknownclients'. On\n"
+            + "       subsequent connections, the client will be rejected if the fingerprint has changed.\n"
+            + "   - insecure-ca-or-tofa: A combination of ca and tofa: If the client presents a certificate\n"
+            + "       signed by a trusted CA, it will be accepted. If it is self-signed, it\n"
+            + "       will be allowed only if it's the first certificate this node has seen for that host.\n"
+            + "   - insecure-record: Any client can connect and the fingerprint of their certificate will be\n"
+            + "       added to the 'tlsknownclients' file.\n",
+        PropertyValidator.anyOfIgnoreCase(
+            "whitelist",
+            "ca",
+            "ca-or-whitelist",
+            "tofu",
+            "insecure-tofa",
+            "ca-or-tofu",
```

Comment:
this, for one, isn't documented above... is the intent to provide usage on all of these options? It'll be hard for people to know they're there if they're not documented...

Reply:
tried to update them ... not sure I was successful.

Reply:
Ok, it turns out this is probably a bigger issue.
I believe TOFU should be used for clients, TOFA is for servers.
Really - the Valid cases for a server param shouldn't have TOFU in them at all.... but that's where this becomes bigger.
Generally - Orion needs a sweep out - don't want to do it here.